builds/e2e/e2e.yaml

```
@@ -0,0 +1,87 @@
+# End-to-end tests should run after new docker images and edgelet packages are
+# built, but 'build completion triggers' are not yet supported in YAML. Set to
+# 'none' here and override in the online editor.
+trigger: none
+pr: none
+
+jobs:
+
+################################################################################	
+  - job: linux_arm32v7
+################################################################################	
+    displayName: Linux arm32v7
+
+    pool:
+      name: Azure-IoT-Edge-Core
+      demands: rpi3-e2e-tests
```

refer to https://github.com/Azure/iotedge/blob/338cc3e53ddd3afed841aa00cc6cac6fd5345fde/builds/e2e/stresstest.yaml#L101. 

```suggestion
      demands:
        - Agent.OS -equals Linux
        - Agent.OSArchitecture -equals ARM
        - run-e2e-tests -equals true
        - agent-osbits -equals 32
```

---

Using the `Agent.*` variables is fine, as is using the custom `agent-osbits`. But I'm intentionally avoiding the `run-e2e-tests` demand since the existing end-to-end tests use it, and I don't want the old and new systems to get mixed together. (Technically none of the Raspberry Pi agents use that demand anyway so it wouldn't conflict, but I just want to avoid confusion while there are two different end-to-end systems in place.)

---

Got it.